core/src/main/java/com/linecorp/armeria/client/retry/RetryRule.java

```diff
@@ -255,6 +256,26 @@ static RetryRule onException(Backoff backoff) {
         return builder().onException().thenBackoff(backoff);
     }
 
+    /**
+     * Returns a newly created {@link RetryRule} that retries with
+     * {@linkplain Backoff#ofDefault() default backoff} on {@link TimeoutException}.
+     * Note that this rule should be used carefully because it reties regardless of
+     * <a href="https://developer.mozilla.org/en-US/docs/Glossary/Idempotent">idempotency</a>.
+     */
+    static RetryRule onTimeoutException() {
```

Comment:
Should we add `@UnstableApi` for the new public APIs?

Reply:
✅ updated